source/features/edit-comments-faster.tsx

```diff
@@ -1,21 +1,22 @@
 import React from 'dom-chef';
-import select from 'select-dom';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
 import PencilIcon from 'octicon/pencil.svg';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
 function init(): void {
-	const menuItems = select.all('details .js-comment-edit-button:not(.rgh-edit-comment)');
+	observe('details details-menu:not([src]) .js-comment-edit-button:not(.rgh-edit-comment)', {
+		add(item) {
+			item.classList.add('rgh-edit-comment');
 
-	for (const item of menuItems) {
-		item.classList.add('rgh-edit-comment');
-
-		const button = item.cloneNode();
-		button.append(<PencilIcon/>);
-		button.classList.replace('dropdown-item', 'timeline-comment-action');
-		item.closest('details')!.before(button);
-	}
+			const button = item.cloneNode();
+			button.append(<PencilIcon/>);
+			button.classList.replace('dropdown-item', 'timeline-comment-action');
+			item.closest('.js-minimizable-comment-group')!.querySelector('.js-comment-header-reaction-button')!.after(button);
```

Comment:
Indeed. But the feature could now be shorter, if you want to try this:

```js
observe('js-comment-header-reaction-button:not(.rgh-edit-comment)', {
	add(reactions) {
		reactions.after(
			<button className="timeline-comment-action" etc etc>
				<PencilIcon/>
			</button>
		);
	}
});

Reply:
> ```js
> reactions.classList.add('rgh-edit-comment');
> ```

Is this needed? No code use it and I don't think it needs to customized

Reply:
Instinctive and works well 👍